src/script/interpreter.cpp

```diff
@@ -15,6 +15,9 @@
 
 typedef std::vector<unsigned char> valtype;
 
+static_assert(static_cast<uint64_t>(std::numeric_limits<int64_t>::max()) == UINT64_C(0x7FFFFFFFFFFFFFFF));
```

Comment:
Unfortunately the ' symbol did not work inside the UINT64_C macro

Reply:
That's a shame.